tensorboard/webapp/notification_center/_views/notification_center_component.ng.html

```diff
@@ -48,7 +48,13 @@ <h3 class="title">{{ notification.title }}</h3>
           class="content"
           [markdown]="notification.content"
         ></markdown-renderer>
-        <div class="extended-buttons">READ FULL</div>
+        <div class="extended-buttons">
+          <a
+            *ngIf="notification.fullNoteLink"
+            href="{{notification.fullNoteLink}}"
```

Comment:
few things:
- `target="_blank"` so we do not navigate away.
- `rel="noreferrer noopener"` so we get less creepy analytics.